test_project/test_app/tests.py

```diff
@@ -39,6 +42,15 @@ def test_create_global_permission(self):
         perm = Permission.objects.filter(codename=GlobalPermission.codename)
         self.assertTrue(perm, 'The permission should exist, but it doesnt.')
 
+    def test_invalid_default_group(self):
+        del settings.DEFAULT_GROUPS_MODULE
+
+        create_groups.Command().handle()
+        #TODO: hide and test stdout: self.assertEquals("DEFAULT_GROUPS_MODULE setting is not set!", output)
```

Comment:
Do we want create_groups.Command().handle() to simply print the error?

Or do we want it to throw an exception? For that [the exception in create_groups.Command().handle()](https://github.com/serioeseGmbH/serious-django-permissions/blob/master/serious_django_permissions/management/commands/create_groups.py#L26) would have to be changed to something like:
```
if not settings.DEFAULT_GROUPS_MODULE:
        raise AttributeError("DEFAULT_GROUPS_MODULE setting is not set!")


        else:
            groups_module = settings.DEFAULT_GROUPS_MODULE
```

Reply:
Changed in https://github.com/serioeseGmbH/serious-django-permissions/commit/3bf6054150e5fe1899fbdc9138fc8e2e9e189f38